types/three/examples/jsm/renderers/common/Color4.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Color, ColorRepresentation } from '../../../../src/math/Color.ts';
+
+export type Color4Representation = ColorRepresentation | Color4;
+
+export default class Color4 extends Color {
+    constructor(r: number, g: number, b: number, a?: number);
+
+    set(...args: [color: Color4Representation] | [r: number, g: number, b: number, a?: number]): this;
```

Comment:
It doesn't seem like `Color4.set` can handle a `Color4` correctly. Maybe we should just stick with `ColorRepresentation` instead of `Color4Representation` here?

Reply:
happy to try